core/domain/email_manager_test.py

```diff
@@ -114,6 +114,41 @@ def test_email_to_admin_is_sent_correctly(self):
             self.assertIn('Dummy Body', messages[0].html.decode())
 
 
+class DummyMailTest(test_utils.GenericTestBase):
+    """Test that emails are correctly sent to the testing email id."""
+
+    def test_sending_emails(self):
+        dummy_system_name = 'DUMMY_SYSTEM_NAME'
+        dummy_system_address = 'dummy@system.com'
+        dummy_receiver_address = 'admin@system.com'
+        dummy_email_body = 'This is a test mail from Oppia'
```

Comment:
Use a username that looks like a reasonable username, not "Oppia".

Reply:
Done.